prebid-mobile/prebid-mobile-pbs.md

```diff
@@ -10,37 +10,43 @@ nav_section: prebid-mobile-pbs
-Prebid Server is an open source project.  [The source code is hosted under the Prebid organization on Github](https://github.com/prebid/prebid-server).
+{: .alert.alert-info :}
+If this is your first time working with header bidding, we recommend that you read [What is Prebid?]({{site.github.url}}/overview/intro.html) before diving into Prebid Mobile.
 
 * TOC
 {:toc}
 
-## Step 1. Register for a Prebid Server account
+## Set Up Prebid Server
+
+Before you begin using Prebid Mobile in your apps, you need to prepare your end-to-end system. The first step is to set up your Prebid Server host. Prebid Mobile works in conjunction with Prebid Server to request and receive bids. At this time you have three options when selecting your Prebid Server host:
+
+-   Register with AppNexus
+-   Register with Rubicon Project
+-   Set up your own Prebid Server host
+
+### Register with AppNexus
+
+-   Go to the [Prebid Server sign-up page](https://prebid.adnxs.com) and click the button to sign up.
+-   You'll receive an account ID and will be able to set up server-side bidder configurations.
 
-- Go to the [Prebid Server sign-up page](https://prebid.adnxs.com) and click the button to sign up.
+### Register with Rubicon Project
 
-- Fill out the form details, including your email address.
+Send an email to prebidserver@rubiconproject.com. While you wait for a verification email, you can continue with the steps below.
 
-- When approved, you will receive an email with your assigned `accountId`. You will need this for configuring Prebid Mobile to use Prebid Server.
+### Implement Your Own Prebid Server Host
 
-## Step 2. Setup your demand partner configurations on Prebid Server
+Prebid Server is an open source project. This allows you to host your own implementation of Prebid Server. The source code is available under the [Prebid organization on Github](https://github.com/prebid/prebid-server).
 
-- Go to the [Prebid Server configs page](https://prebid.adnxs.com/configs/) and click the button to add a new demand source config.
-- Each demand source config maps to an ad unit. For every ad unit in your app, you will have to create a new demand source config.
-- Take note of the `configId` generated by PBS. You will use it later in your iOS or Android app's implementation of Prebid Mobile SDK
+See the [Prebid Server docs on Github](https://github.com/prebid/prebid-server/tree/master/docs) for more information on setting up your own server host.
 
-A demand source config is a list of bidders that will participate in the auction for an ad slot written in JSON. For each JSON object in the array, a `"bidder"` field is required as well as a `"params"` object for demand source specific parameters.
+## Configure Prebid Server
 
-Example config with AppNexus demand source:
+After you've registered with your chosen Prebid Server host, configure Prebid Server for each bidder as required by the host. The configuration will be in the form of a JSON structure, similar to this:
```

Comment:
The language here makes it sound like this is some kind of one-time configuration that needs to be done when deploying PBS.  Let's be clear that users need to create one or more unique Prebid Server Configurations where each Configuration is a list of bidders and their respective parameters.